mlxtend/regressor/linear_regression.py

```diff
@@ -21,13 +21,18 @@ class LinearRegression(_BaseModel, _IterativeModel, _Regressor):
     ------------
     eta : float (default: 0.01)
         solver rate (between 0.0 and 1.0)
+    method : string (default: 'direct')
+        Which method do you want to use to solve it.
+        For Gradient Descent, use 'sgd', for QR decomposition method, use
+        'qr', for SVD method, use 'svd'and for direct analytical method,
```

Comment:
add a whitespace after 'svd'

Reply:
I suggest the following rewording:

For gradient descent-based optimization, use 'sgd' (see `'minibatch'` parameter for further options). Otherwise, if `'direct'` (default), the analytical method is used. For alternative, numerically more stable solutions, use either `'qr'` (QR decomopisition) or `'svd'` (Singular Vector Decomposition).

Reply:
Done

Reply:
Can't see the changes, yet, but you probably haven't pushed the latest commit to the PR.